metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -269,7 +269,10 @@ public void testSnapshotSaveAndLoad() throws Throwable {
                 reader = logEnv.waitForSnapshot(snapshotLogOffset);
                 SnapshotReader<ApiMessageAndVersion> snapshot = createSnapshotReader(reader);
                 assertEquals(snapshotLogOffset, snapshot.lastContainedLogOffset());
-                checkSnapshotContents(fooId, brokerEpochs, snapshot);
+                checkSnapshotContent(
```

Comment:
Nitpick: I think in this case this would be more readable on 2 lines rather than 4.

Reply:
Done.